aws/resource_aws_cloudfront_distribution_test.go

```diff
@@ -75,29 +75,6 @@ func testSweepCloudFrontDistributions(region string) error {
 	return nil
 }
 
-func TestAccAWSCloudFrontDistribution_importBasic(t *testing.T) {
```

Comment:
Moved import state verification to all acceptance tests to ensure we are consistently reading into the Terraform state from all API responses.